omx_components/src/mfx_omx_vdec_component.cpp

```diff
@@ -2033,7 +2033,12 @@ void MfxOmxVdecComponent::MainThread(void)
                             }
                         }
                         else
+                        { 
+                            MFX_OMX_LOG_INFO_IF(g_OmxLogLevel, "Failed to get buffer, maybe all buffers are allocated!");
```

Comment:
There is no error, branch "else" is correct in this way.
We can get here when we decoded full bitstream (which we've got from framework), and we are going to wait new part of bitstream from framework to continues decoding

Reply:
Revert changes back to initial PR and also add brief error handing, please check if this is fine?